lib/constants.ts

```diff
@@ -1,13 +1,13 @@
 import { Address, zeroAddress } from "viem";
 
 export const PAYMENT_ESCROW =
-  "0x1901D7DFb5614F85D805C3adb987dB566B1d40Ed" as Address;
-export const USDC = "0x036CbD53842c5426634e7929541eC2318f3dCF7e" as Address;
+  "0x1901D7DFb5614F85D805C3adb987dB566B1d40Ed" as const satisfies Address;
```

Comment:
ugh this is grosser though with more boiler plate ts nonsense

Reply:
lol reverted